Orange/widgets/unsupervised/tests/test_owdistancemap.py

```diff
@@ -0,0 +1,50 @@
+# Test methods with long descriptive names can omit docstrings
+# pylint: disable=missing-docstring
+import random
+import numpy as np
+from Orange.data import Table
+from Orange.misc.flagged_data import FLAGGED_SIGNAL_NAME, FLAGGED_FEATURE_NAME
+from Orange.distance import Euclidean
+from Orange.widgets.unsupervised.owdistancemap import OWDistanceMap
+from Orange.widgets.tests.base import WidgetTest
+
+
+class TestOWDistanceMap(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWDistanceMap)
+        self.iris = Table("iris")
+        self.iris_distances = Euclidean(self.iris)
+
+    def test_outputs(self):
+        self.send_signal("Distances", self.iris_distances)
+
+        # check selected data output
+        self.assertIsNone(self.get_output("Data"))
+
+        # check flagged data output
+        flagged = self.get_output(FLAGGED_SIGNAL_NAME)
+        self.assertEqual(0, np.sum([i[FLAGGED_FEATURE_NAME] for i in flagged]))
+
+        # select data points
+        points = random.sample(range(0, len(self.iris)), 20)
```

Comment:
I prefer deterministic tests. Randomly choose some indices instead of choosing some indices at random. (https://xkcd.com/221/).